Rubberduck.Resources/Inspections/InspectionResults.de.resx

```diff
@@ -332,4 +332,46 @@
   <data name="StepIsNotSpecifiedInspection" xml:space="preserve">
     <value>Step ist nicht angegeben</value>
   </data>
+  <data name="SheetAccessedUsingStringInspection" xml:space="preserve">
+    <value>Statisch erreichbares Blatt kann mit dem Code-Namen referenziert werden.</value>
```

Comment:
Vielleicht besser _Tabellenblatt_. Neue Blätter werden in deutschem Excel einfach mit _Tabelle{Nummer}_ bezeichnet.